app/src/main/java/explorer/city/com/cityexplorer/view/activity/MainActivity.kt

```diff
@@ -18,10 +18,7 @@ import java.util.concurrent.TimeUnit
 
 class MainActivity : LifecycleActivity() {
 
-    companion object {
-        val TAG: String = "MainActivity"
-    }
-
+    val TAG: String = "MainActivity"
```

Comment:
🤔 I read that it were possible. `companion` seems to be the replacement for `public static`. It seems to be the case that if you want `private static` you just use `private val`